source/2019-12-09-countdown-to-the-new-year-ember-angle-brackets-codemod.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Countdown to The New Year - Ember Angle Brackets Codemod
+author: Ryan Mark
+tags: 2019, Addons, DecEmber
+alias: 2019-12-09-countdown-to-the-new-year-ember-angle-brackets-codemod.md
+responsive: true
+---
+
+**This is the ninth in our DecEmber series–"Countdown to The New Year: 31 Days of Ember Addons". We plan to highlight a new addon each day until the new year, and we hope you'll join us for the fun!**
+
+## Day 9
+
+Until recently, converting Ember templates to use Angle Bracket invocation was a tiresome and manual task. With the latest version of [ember-angle-brackets-codemod](https://www.emberobserver.com/addons/ember-angle-brackets-codemod), you get a smarter, more complete codemod to change your templates in a flash!
+
+READMORE
+
+### What It Does
+
+Before Angle Bracket invocation, nearly every template construct in Ember was invoked using Handlebars `{{doubleCurly}}` syntax. This made differentiating between components, helpers, and plain values incredibly difficult.  Angle Brackets alleviate most of this ambiguity.  But what if you wanted to change every double curly component invocation to Angle Brackets?  That task could take forever (especially if you cant reliably tell the difference between everything in your templates)!  The Angle Bracket Codemod eases the transition in a few simple steps:
```

Comment:
```suggestion
Before Angle Bracket invocation, nearly every template construct in Ember was invoked using Handlebars's `{{doubleCurly}}` invocation. This made it difficult to quickly differentiate between components, helpers, and plain values.  The `<AngleBracket />` syntax alleviates quite a bit of this ambiguity.  But what if you wanted to update every component invocation to use Angle Bracket syntax?  That task could take forever, especially if you can't reliably tell the difference between everything in your templates!  The `ember-angle-brackets-codemod` addon helps to ease the transition in a few simple steps:
```
Suggested refactor for grammatical clarity. May need to add some `alex ignore` statements in here. 
FWIW, we have tried to be consistent by using "angle bracket syntax" and "curly brace syntax" IIRC, @jenweber please correct if I've misremembered.

Reply:
@MelSumner I agree with the suggestion but I don’t think this an addon - it doesn’t install or get used like one or stay as a dependency to the project/app